conda_build/environ.py

```diff
@@ -47,6 +47,10 @@ def get_git_build_info(src_dir, git_url, expected_rev):
     env = os.environ.copy()
     d = {}
     git_dir = join(src_dir, '.git')
+    if not isinstance(git_dir, str):
```

Comment:
Not only is the problem unique to windows, but only to Python < 3 (from what I can tell, this has to do with Win9x support in earlier versions of Python)